src/blog/dark-mode-with-style-dictionary/index.mdx

```diff
@@ -0,0 +1,1352 @@
+
+<ThemedImage style={{width: '48%', float: 'left'}}>
+
+![Project view shows actual folder structure](images/as-project-light.png)
+![Project view shows actual folder structure](images/as-project-dark.png)
+
+</ThemedImage>
+
+<ThemedImage style={{width: '48%', float: 'right'}}>
+
+![Android view organizes files to reduce clutter](images/as-android-light.png)
+![Android view organizes files to reduce clutter](images/as-android-dark.png)
+
+</ThemedImage>
+
+<div style={{clear: 'both'}} />
+
+### Graphics in Android
+
+Android has support for vector graphics with its [Android Vector Drawable](https://developer.android.com/guide/topics/graphics/vector-drawable-resources) resource that shares a lot with SVG. Here is an example of Android Vector graphics format and SVG:
+
+```xml
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+  android:width="128dp"
+  android:height="128dp"
+  android:viewportWidth="128"
+  android:viewportHeight="128">
+  <path
+    android:strokeColor="#0b968f"
+    android:strokeWidth="8"
+    android:strokeLineJoin="round"
+    android:pathData="M89.75 49.85L115 35.15v58.79l-50.5 29.4V90.93"/>
+</vector>
+```
+
+```xml
+<svg width="128" height="128" viewBox="0 0 128 128" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path
+    stroke="#0b968f"
+    stroke-width="8"
+    stroke-linejoin="round"
+    d="M89.75 49.847L115 35.1493V93.9403L64.5 123.336V90.9254" />
+</svg>
+```
+
+Very similar to SVG on the web, we can:
+1. Provide alternative graphics based on color mode with resource qualifiers
+1. Use design tokens (color resources in Android, CSS custom properties in Web) inside the graphic
+
+Here is the same Android vector drawable with references to resources. If the colors referenced in this graphic have light and dark qualified values, those will be applied appropriately. No need to create a separate dark version of this graphic. 
+
+```xml
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+  android:width="128dp"
+  android:height="128dp"
+  android:viewportWidth="128"
+  android:viewportHeight="128">
+  <path
+    android:strokeColor="@color/color_brand_primary"
+    android:strokeWidth="@dimen/size_border_width_large"
+    android:strokeLineJoin="round"
+    android:pathData="M89.75 49.85L115 35.15v58.79l-50.5 29.4V90.93"/>
+</vector>
+```
+
+What we will do is take graphics written in SVG defined in our Style Dictionary package and convert them to Android vector drawable files. Android also has support for raster based graphics like PNG as well so we could convert our graphics to PNG, but because we can use a vector based format we will go with that. We will be using an NPM library, [svg2vectordrawable](https://www.npmjs.com/package/svg2vectordrawable), to translate an SVG file into Android Vector Drawable format.
```

Comment:
What we will do is take graphics written in SVG > We will take graphics written in SVG